tools/rna_tools/targetfinder/TargetFinder_threads.xml

```diff
@@ -0,0 +1,57 @@
+<tool id="targetfinder_threads" name="targetfinder_threads" version="0.1.0">
```

Comment:
Why do we have 2 tools? There is not real difference from a user point of view right?